src/Components/NotFoundPage.test.jsx

```diff
@@ -0,0 +1,11 @@
+import { render } from '@testing-library/react';
+
+import NotFoundPage from './NotFoundPage';
+
+test('renders NotFoundPage', () => {
+  const { queryByText } = render(
+    <NotFoundPage />,
+  );
```

Comment:
```suggestion
  const { queryByText } = render((
    <NotFoundPage />
  ));
```

괄호를 두번 감싸주시면 콤마를 없앨 수 있어요.